c2corg_ui/templates/waypoint/view.html

```diff
@@ -42,7 +42,7 @@
 </%block>\
 
 <header class="view-details-title">
-  <span class="heading">${locale['title']} <label class="badge" translate>waypoint</label></span>
+  <span class="heading"><h1>${locale['title']} <label class="badge" translate>waypoint</label></h1></span>
```

Comment:
Do you really need to have both ``<span class="heading">`` and ``<h1>``?
If the "heading" class is required, what about simply writing
```
<h1 class="heading">
```
?